core/setup.py

```diff
@@ -56,7 +56,7 @@
         "agate>=1.6,<1.6.4",
         "click>=7.0,<9",
         "colorama>=0.3.9,<0.4.5",
-        "hologram==0.0.14",
+        "hologram==0.0.15",
```

Comment:
I think we should aim to include this in a v1.1 patch release (since really this is just bumping the upper bound on the `jsonschema` dependency). For anyone who's _hypothetically_ pinned to `hologram==0.0.14`, that could be a breaking change.

Either in this PR or in the backport PR, could we make this:
```suggestion
        "hologram>=0.0.14,<=0.0.15",
```

Reply:
Good call out Jeremy!